sdk/storage/Azure.Storage.DataMovement.Blobs/README.md

```diff
@@ -40,7 +40,178 @@ az storage account create --name MyStorageAccount --resource-group MyResourceGro
 ### Authenticate the client
 In order to interact with the Data Movement library you have to create an instance with the TransferManager class.
 
-TODO: Insert code snippet here.
+```C# Snippet:UploadSingle_ConnectionStringAsync
```

Comment:
Can you add headers here, for each of the samples?
 `### Start Single Transfer from Local File to Block Blob`

Reply:
Also could you move all these snippets down to where the Examples are?

We can possibly leave the sample on how to instantiate a TransferManager object here.